stats/rpc_constants.textpb

```diff
@@ -0,0 +1,620 @@
+    ]
+  }
+  tag_keys: [
+    "method" # Tag value will be the name of server RPC method/service
+  ]
+}
+
+view_descriptor {
+  name: "grpc.io/server/server_elapsed_time/distribution_cumulative"
+  description: "Server elapsed time in milliseconds on server RPC side"
+  measurement_descriptor_name: "grpc.io/server/server_elapsed_time"
+  distribution_aggregation: {
+    bucket_bounds: [
+      0.0, 1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 8.0, 10.0, 13.0, 16.0, 20.0, 25.0, 30.0,
+      40.0, 50.0, 65.0, 80.0, 100.0, 130.0, 160.0, 200.0, 250.0, 300.0, 400.0, 500.0, 650.0,
+      800.0, 1000.0, 2000.0, 5000.0, 10000.0, 20000.0, 50000.0, 100000.0
+    ]
+  }
+  tag_keys: [
+    "method" # Tag value will be the name of server RPC method/service
+  ]
+}
+
+view_descriptor {
+  name: "grpc.io/server/request_bytes/distribution_cumulative"
+  description: "Request bytes of server RPCs"
+  measurement_descriptor_name: "grpc.io/server/request_bytes"
+  distribution_aggregation: {
+    bucket_bounds: [
+      0.0, 1024.0, 2048.0, 4096.0, 16384.0, 65536.0, 262144.0, 1048576.0, 4194304.0,
+      16777216.0, 67108864.0, 268435456.0, 1073741824.0, 4294967296.0
+    ]
+  }
+  tag_keys: [
+    "method" # Tag value will be the name of server RPC method/service
+  ]
+}
+
+view_descriptor {
+  name: "grpc.io/server/response_bytes/distribution_cumulative"
+  description: "Response bytes of server RPCs"
+  measurement_descriptor_name: "grpc.io/server/response_bytes"
+  distribution_aggregation: {
+    bucket_bounds: [
+      0.0, 1024.0, 2048.0, 4096.0, 16384.0, 65536.0, 262144.0, 1048576.0, 4194304.0,
+      16777216.0, 67108864.0, 268435456.0, 1073741824.0, 4294967296.0
+    ]
+  }
+  tag_keys: [
+    "method" # Tag value will be the name of server RPC method/service
+  ]
+}
+
+view_descriptor {
+  name: "grpc.io/server/uncompressed_request_bytes/distribution_cumulative"
+  description: "Uncompressed Request bytes of server RPCs"
+  measurement_descriptor_name: "grpc.io/server/uncompressed_request_bytes"
+  distribution_aggregation: {
+    bucket_bounds: [
+      0.0, 1024.0, 2048.0, 4096.0, 16384.0, 65536.0, 262144.0, 1048576.0, 4194304.0,
+      16777216.0, 67108864.0, 268435456.0, 1073741824.0, 4294967296.0
+    ]
+  }
+  tag_keys: [
+    "method" # Tag value will be the name of server RPC method/service
+  ]
+}
+
+view_descriptor {
+  name: "grpc.io/server/uncompressed_response_bytes/distribution_cumulative"
+  description: "Uncompressed Response bytes of server RPCs"
+  measurement_descriptor_name: "grpc.io/server/uncompressed_response_bytes"
+  distribution_aggregation: {
+    bucket_bounds: [
+      0.0, 1024.0, 2048.0, 4096.0, 16384.0, 65536.0, 262144.0, 1048576.0, 4194304.0,
+      16777216.0, 67108864.0, 268435456.0, 1073741824.0, 4294967296.0
+    ]
+  }
+  tag_keys: [
+    "method" # Tag value will be the name of server RPC method/service
+  ]
+}
+
+# Census defined RPC client IntervalViewDescriptors.
```

Comment:
I think this should have a space, i.e., "client interval ViewDescriptors", because we don't have a separate `IntervalViewDescriptor` message.